dev-docs/modules/userid-submodules/lotame.md

```diff
@@ -9,11 +9,11 @@ example: '"e4b9..."'
 ---
 
 
-[Lotame Panorama](https://www.lotame.com/panorama/) is a suite of data-enrichment solutions for digital advertising that empowers marketers, agencies, publishers and media companies to transform consumer personas into addressable audiences. At the heart of Lotame Panorama is the Panorama ID, a people-based identifier powered by deterministic and probabilistic data, available across the cookie-challenged web and all browsers.
+[Lotame Panorama ID™](https://www.lotame.com/panorama-id/) is a pseudonymous ID that represents devices for the purposes of audience enrichment and campaign activation. It is powered by the Lotame Panorama Graph, which is built on hashed emails and browser and device data and combined with machine learning and predictive models to estimate the likelihood that a group of devices may be used by the same user or are in the same household. Panorama ID is among the cookieless leaders in terms of publisher adoption and bid stream coverage.
```

Comment:
```suggestion
[Lotame Panorama ID™](https://www.lotame.com/panorama-id/) is a pseudonymous ID that represents devices for the purposes of audience enrichment and campaign activation. It is powered by the Lotame Panorama Graph, which is built on hashed emails and browser and device data and combined with machine learning and predictive models to estimate the likelihood that a group of devices may be used by the same user or are in the same household.
```

This is the technical documentation :smile:  We try to limit marketing speech as much as possible.